openfisca_france_local/aides_jeunes_reform.py

```diff
@@ -0,0 +1,269 @@
+# -*- coding: utf-8 -*-
+
+import yaml
+import operator
+
+from pathlib import Path
+from numpy.core.defchararray import startswith
+import numpy as np
+
+from openfisca_france.model.base import *
+from openfisca_core import reforms
+from openfisca_core.periods import Period
+
+
+from openfisca_france.model.caracteristiques_socio_demographiques.demographie import RegimeSecuriteSociale
+from openfisca_france.model.caracteristiques_socio_demographiques.demographie import GroupeSpecialitesFormation
+from openfisca_france.model.prestations.education import TypesScolarite, TypesClasse
+from openfisca_core.populations.population import Population
+
+operations = {
+    '<': operator.lt,
+    '<=': operator.le,
+    '>': operator.gt,
+    '>=': operator.ge,
+}
+
+
+def is_age_eligible(individu, period, condition):
+
+    condition_age = condition['value']
+    individus_age = individu('age', period.first_month)
```

Comment:
```suggestion
    individus_age = individu('age', period)
```

Pour alléger le code, je ferais l'hypothèse que `period` est un mois.